.tools/readmes/config.py

```diff
@@ -32,27 +32,40 @@
         }
     },
     "Go": {
+        1: {
+            "base_folder": "go",
+            "service_folder": 'go/{{service["name"]}}',
+            "deprecated": "true",
```

Comment:
People are very touchy about the word deprecated so I would choose something else. Otherwise, you risk someone thinking you actually mean the SDK is officially deprecated, which it isn't.

Reply:
Suggestions?

Reply:
deprecated -> unsupported